### DISCUSSION TITLE
求问一下离线下载中的 115 cloud 运作方式

### DISCUSSION THREAD

**Discussion by Venompool888** [AlistGo/alist#7044] [Q&A] [answered]
已经阅读了 https://alist.nn.ci/zh/guide/advanced/offline-download.html#_5-115-cloud
但是还有一些疑问
· 这个方式是把磁力链接先解析到115网盘，然后再下载到alist所指目录下么？
· 既然是这样子的话，那意味着就必须登录115账号。可是我的Alist上面挂载了两个115账号，那我这个磁类链接怎样才能指定到具体的哪一个115账号上进行离线呢？还是说会登录的所有账号，都对这一个磁力链接进行离线？
· 可以只是把此类链接解析到115网盘上，而非传输到alist下目录吗？

**Comment by Simonzhang8** [comment]
你在哪个文件夹下点击离线下载，文件就下载到哪个文件夹里。
你挂载两个115应该是挂载到不同的目录下的吧，那么你在哪个目录下的文件夹操作，就是离线下载到对应的那个账号里面

**Comment by anwen-anyi** [comment]
1. 这个是调用115官方里面的离线下载接口

2. 可是我的Alist上面挂载了两个115账号，那我这个磁类链接怎样才能指定到具体的哪一个115账号上进行离线呢？
   - 你想把文件离线到哪儿，你就打开那个目录，他就会自动离线到该目录下，（因为是115官方接口 需要会员）

3. 那我这个磁类链接怎样才能指定到具体的哪一个115账号上进行离线呢？
   - 你自己挂载的115账号，你不知道那个目录是那个账号的吗？

4. 可以只是把此类链接解析到115网盘上，而非传输到alist下目录吗？
   - 本来就是直接解析到115网盘里面 并非alist目录下

具体的情况你自己测试一下就懂了

**Reply by Venompool888** [reply]
> 1. 这个是调用115官方里面的离线下载接口
> 2. 可是我的Alist上面挂载了两个115账号，那我这个磁类链接怎样才能指定到具体的哪一个115账号上进行离线呢？
>    
>    * 你想把文件离线到哪儿，你就打开那个目录，他就会自动离线到该目录下，（因为是115官方接口 需要会员）
> 3. 那我这个磁类链接怎样才能指定到具体的哪一个115账号上进行离线呢？
>    
>    * 你自己挂载的115账号，你不知道那个目录是那个账号的吗？
> 4. 可以只是把此类链接解析到115网盘上，而非传输到alist下目录吗？
>    
>    * 本来就是直接解析到115网盘里面 并非alist目录下
> 
> 具体的情况你自己测试一下就懂了

可是我在 alist 里别的不是 115 网盘的目录下也可以有 115 cloud 这个选项

**Reply by anwen-anyi** [reply, marked as answer]
> 可是我在 alist 里别的不是 115 网盘的目录下也可以有 115 cloud 这个选项

能看到是能看到，但是添加进去不能用，文档也提示啦
![image](https://github.com/user-attachments/assets/1ca750fe-660f-4624-9bae-276d84002b83)
